addons/spreadsheet_dashboard/tests/test_dashboard_share.py

```diff
@@ -0,0 +1,41 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from .common import DashboardTestCommon
+from odoo.exceptions import AccessError
+
+EXCEL_FILES = [
+    {
+        "content": '<Types xmlns="http://schemas.openxmlformats.org/package/2006/content-types"/>',
+        "path": "[Content_Types].xml",
+    }
+]
+
+class DashboardSharing(DashboardTestCommon):
+    def test_share_url(self):
+        dashboard = self.create_dashboard()
+        share_vals = {
+            "spreadsheet_data": dashboard.spreadsheet_data,
+            "dashboard_id": dashboard.id,
+            "excel_files": EXCEL_FILES,
+        }
+        url = self.env["spreadsheet.dashboard.share"].action_get_share_url(share_vals)
+        share = self.env["spreadsheet.dashboard.share"].search(
+            [("dashboard_id", "=", dashboard.id)]
+        )
+        self.assertEqual(url, share.full_url)
+        self.assertEqual(share.dashboard_id, dashboard)
+        self.assertTrue(share.excel_export)
+
+    def test_can_create_own(self):
```

Comment:
Interesting to add a test where a use tries to create a share with another `create_uid`

Reply:
@LucasLefevre I didn't get "create a share with another create_uid". Isn't `create_uid` automatically filled in when creating a new share?